index.js

```diff
@@ -1,26 +1,41 @@
 
-function calculate(calculationSteps) {
-  // multiplies two numbers when one is stringified
+function calculateNumberandString(){}
+function calculateNullAsZero(){}
+function calculateIgnoreUndefine(){}
+function calculateIgnoreEmptyString(){}
+function calculateNonNumericIgnore(){}
+
+
+
 
 
 
-  // treats null values as zeroes and includes them in the calculation
 
 
-  // ignores undefined and empty string values and continues operation
 
 
-  // ignores non-numeric values  and continues operation
 
 
-  // returns NaN if the array does not represent a proper operation
 
+
+
+function calculate(calculationSteps) {
   switch (calculationSteps) {
     case 'numbers':
+      // multiplies two numbers when one is stringified
```

Comment:
Make sure that you are removing all pseudo code or notes, any unused code. "Code Clean up"